docs/faq/helium-network.mdx

```diff
@@ -13,70 +13,66 @@ slug: /faq/helium-network
 
-### Can I connect my phone or computer to Hotspots?
-
-Yes. As of early 2022 the community has started to roll out
-[4G/LTE and 5G coverage on the Helium blockchain](https://www.helium.com/5G).
-
-### If I connect the Hotspot to my Wifi, am I breaking the terms of service with my Internet provider?
-
-The Helium Network is not in the position or business of giving legal advice. In general, users
-should check the terms with individual providers and comply with applicable law.
-
-### Does Helium partner with Internet providers?
-
-Yes. In fact, Helium recently partnered with DISH, a nationwide provider of internet connectivity,
-to expand 5G coverage.
-[You can learn more about this news here](https://ir.dish.com/news-releases/news-release-details/dish-partners-helium-leverage-helium-networks-blockchain-model).
-
 ### Does Helium replace the Wi-Fi in my house?
 
-No. Hotspots create a Long-Fi radio signal and routes packets from low-power devices in your area
-that use LoRaWAN, a wireless protocol, and have been deployed to the Helium Network. Typically these
+No. Hotspots receive radio signals and route packets from low-power devices in your area that use
+LoRaWAN, a wireless protocol, and have been deployed to the Helium Network. Typically these
 are devices such as GPS trackers, environmental sensors, weather meters, etc., that only need to
 transmit and share small bits of information. The Hotspot uses your existing Internet (via WiFi or
 Ethernet) to deliver the data packets sent by devices. It does not replace internet or cellular
-service for regular devices like computers and smartphones.
+service for regular devices like computers and smartphones
 
-### Why is the Helium Network called The People’s Network?
+### If I connect the Hotspot to my Wifi, am I breaking the terms of service with my Internet provider?
 
-We call it The People’s Network because the people who deploy Hotspots build and operate the
-network. It’s powered by them and owned by them.
+The Helium Network is not in the position or business of giving legal advice. In general, users
+should check the terms with individual providers and comply with applicable law.
 
 ### How far does the Helium Network reach and does it use radio frequency?
 
 Helium’s LoRaWAN technology, the combination of the Helium blockchain and LoRaWAN wireless protocol,
-uses sub-GHz radio frequency. Depending on conditions, Helium LoRaWAN can reach miles and even
-connect devices that are inside buildings unlike GPS and cellular signals.
+uses sub-GHz radio frequency between 860 and 923 MHz. Depending on conditions, Helium LoRaWAN can 
+reach dozens of miles or kilometers and even connect devices that are inside buildings unlike GPS and 
+cellular signals.
 
 ### What kind of cryptocurrency do I earn?
 
-Hotspot owners earn HNT tokens that are unique to the Helium blockchain.
+Hotspot owners earn IOT tokens on the Solana Blockchain
```

Comment:
Lets leave off the mention of Solana here.
We also need to illustrate Mobile Hotspots as entity on the network.

We can link to the IOT and MOBILE pages for more detail.